src/config-parsing/index.ts

```diff
@@ -5,18 +5,16 @@ import reduce from 'lodash/reduce';
 import template from 'lodash/template';
 import { z } from 'zod';
 
-export const secretNamePattern = /^[A-Z][\dA-Z_]*$/;
+export const strictSecretNamePattern = /^[A-Z][\dA-Z_]*$/;
 
-export const secretNameSchema = z
+export const strictSecretNameSchema = z
   .string()
-  .regex(secretNamePattern, `Secret name is not a valid. Secret name must match ${secretNamePattern.toString()}`);
-
-export const secretsSchema = z.record(secretNameSchema, z.string());
+  .regex(
+    /^[A-Z][\dA-Z_]*$/,
```

Comment:
Couldn't this be `strictSecretNamePattern`?

Reply:
Yeah, thx.